app/gui/src/project-view/components/DocumentationEditor.vue

```diff
@@ -41,14 +41,14 @@ function handlePaste(raw: boolean) {
   window.navigator.clipboard.read().then(async (items) => {
     if (!markdownEditor.value) return
     for (const item of items) {
+      if (tryUploadPastedImage(item)) return
       const textType = item.types.find((type) => type === 'text/plain')
       if (textType) {
         const blob = await item.getType(textType)
         const rawText = await blob.text()
         markdownEditor.value.putText(raw ? rawText : transformPastedText(rawText))
-        break
+        return
```

Comment:
Nitpick: Why do you think this is better? I thought the function should start with more typical case (which is text).

Reply:
If both are present, we can assume the text is a fallback.

Reply:
Yes, I think that's fair.